wl/server/SyncThread.java

```diff
@@ -65,14 +65,14 @@ public void run() {
 				Thread.sleep(then - now);
 
 				Utils.log("Waking up for " + (phase == 0 ? "full" : "SQL-only") + " sync.");
-				for (Utils.Databases db : Utils.Databases.values()) Utils.sql(db, "show tables");
+				for (Utils.Databases db : Utils.Databases.values()) Utils.sql(Utils.config(db.db_name), "show tables");
 
 				Utils.log("Backing up the database...");
 				new File("backup").mkdir();
 				Runtime.getRuntime().exec(new String[] {
 				    "bash", "-c",
-				    "mysqldump -u" + Utils.config("databaseuser") + " -p" +
-				        Utils.config("databasepassword") + " -h" + Utils.config("databasehost") +
+				    "mysqldump -u" + Utils.config("addondbeuser") + " -p" +
+				        Utils.config("addondbpassword") + " -h" + Utils.config("databasehost") +
```

Comment:
I think those keys are both misspelled, and it's hard to see whether they are or not.

How about adding some underscores, e.g. addons_db_user, addons_db_password, website_db_user, website_db_password

Reply:
Did that